CreateTables.sql

```diff
@@ -0,0 +1,85 @@
+/* Tabellen löschen */
+/*
+DROP TABLE IF EXISTS bimap.projekt;
+DROP TABLE IF EXISTS bimap.bauart;
+DROP TABLE IF EXISTS bimap.projektverfasser;
+DROP TABLE IF EXISTS bimap.leistung;
+DROP TABLE IF EXISTS bimap.ort;
+DROP TABLE IF EXISTS bimap.bauherr;
+DROP TABLE IF EXISTS bimap.bauartprojekt;
+DROP TABLE IF EXISTS bimap.projektverfasserprojekt;
+DROP TABLE IF EXISTS bimap.leistungprojekt;
+*/
+
+
+/* Tabellen erzeugen */
+CREATE TABLE IF NOT EXISTS projekt
+(
+	id INT, 
```

Comment:
Die id sollte nicht null sein und automatisch durch die Datenbank generiert werden: id INT(11) NOT NULL AUTO_INCREMENT,